test_param_string: Hello World

